packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelFormFields/DefaultValueField.tsx

```diff
@@ -83,33 +83,29 @@ function DefaultValueDateField(props: DefaultValueFieldProps<string>) {
   )
 }
 
-function DefaultValueListField(props: DefaultValueFieldProps<string>) {
+function DefaultValueListField(
+  props: DefaultValueFieldProps<(string | number)[], (string | number)[]>,
+) {
   const {
     onChange,
     value: _value = null,
     columnModel,
     TextFieldProps,
     disabled,
   } = props
-  // The backend returns the default value as a serialized JSON string
-  // Try to deserialize it
   const value = useMemo(() => {
-    if (_value == null || _value == '') {
-      return null
-    }
     try {
-      const parsedValue = JSON.parse(_value) as unknown[]
-      if (columnModel.columnType === ColumnTypeEnum.DATE_LIST) {
+      if (
+        columnModel.columnType === ColumnTypeEnum.DATE_LIST &&
+        Array.isArray(_value)
+      ) {
         // if it's a DATE_LIST, then the values are Unix timestamps in milliseconds
-        // divide by 1000 then parse with dayjs
-        return parsedValue.map(v => {
-          if (typeof v === 'number') {
-            return dayjs(v / 1000)
-          }
-          return dayjs(v as string)
+        // The `MultiValueField` component expects ISO strings to match the JSON Schema 'date-time' format
+        return _value.map(v => {
+          return dayjs(v).toISOString()
         })
       }
-      return parsedValue
+      return _value
```

Comment:
Update the value to match what the actual editor expects -- date-time ISO strings